### PR TITLE
Update FSTextView extension Rename FSE+UITextField file to FSE+UITextView

### PR DESCRIPTION
**Pull request by kruperfone**

### CODE REVIEW

File: Swift/Helpers/FSExtensions/FSE+UITextView.swift [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+//
+//  Created by Kruperfone on 31.07.15.
+//  Copyright (c) 2015 FlatStack. All rights reserved.
+//
+
+import UIKit
+
+class FSTextView :UITextView {
+    
+    private var placeholderLabel:UILabel = UILabel()
```
Comment (vladimirgoncharov) [inline Swift/Helpers/FSExtensions/FSE+UITextView.swift]: I think better to do the variable as private(set) `private(set) let placeholderLabel = UILabel()` that we can configure font or other values in the placeholderLabel in a code another .swift of a file

Comment (kruperfone) [inline Swift/Helpers/FSExtensions/FSE+UITextView.swift]: I make it in analogy with UITextField.

Comment (vladimirgoncharov) [inline Swift/Helpers/FSExtensions/FSE+UITextView.swift]: But it is wrong analogy. :) I explained reason

**Comment by vladimirgoncharov** [comment]
:+1:

**Pull request merged**